stages/org.osbuild.greenboot

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python3
+"""
+Configure greenboot
+
+Update configuration of greenboot in /etc/greenboot/greenbot.conf.
+"""
+
+import fileinput
+import sys
+
+import osbuild.api
+
+
+SCHEMA = """
+"additionalProperties": false,
+"required": ["config"],
+"properties": {
+  "config": {
+    "additionalProperties": false,
+    "description": "greenboot config options",
+    "type": "object",
+    "properties": {
+      "monitor_services": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+      }
+    }
+  }
+}
+"""
+
+
+def main(tree, options):
+    greenboot_conf = options.get("config", {})
+    changes = greenboot_conf.copy()
+    config_file = f"{tree}/etc/greenboot/greenboot.conf"
+
+    # For each of the configured options, find the first non-commented out instance
```

Comment:
I don't see any handling of comments below. Does the file support any?

Reply:
I think it just re-writes every line as the pwquality.conf stage you suggested